index.js

```diff
@@ -13,6 +13,7 @@ module.exports = function (file, opts) {
 
   var blockSize = opts.blockSize || 1024*16
   var codec = opts.codec || {encode: id, decode: id, buffer: true}
+  var flags = opts.flags || 'a+'
```

Comment:
I *think* we removed this in `master` on purpose but I can't remember why. Hmm.

Reply:
I remember! it breaks your delete stuff. unfortunately there is no option that means "open a file for writing and reading and create the file if it does not exist" there is only error if it does not exist

Reply:
Does that mean that we should remove it from here? I don't remember, I can look into it (unless someone else beats me to it).

Reply:
yes, this patch should not change the file open flag, it's not about that.